docs/recent-releases/go-ipfs-0-5/features.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Features
+issueUrl: https://github.com/ipfs/docs/issues/471
+---
+
+# Features
+
+This release is the biggest Go-IPFS release since mid-September 2019. Here are some of the incredible features we've packed in:
+
+## Improved DHT
+
+The distributed hash table (DHT) is how IPFS nodes keep track of who has what data. The DHT implementation has been almost completely rewritten in this release. Providing, finding content, and resolving IPNS records are now all much faster. However, there are risks involved with this update due to the significant amount of changes that have gone into this feature.
+
+### Old and new
+
+The current DHT suffers from three core issues addressed in this release:
+
+- Most peers in the DHT cannot be dialed (e.g., due to firewalls and NATs). Much of a DHT query time is wasted trying to connect to peers that cannot be reached.
+- The DHT query logic doesn't properly terminate when it hits the end of the query and, instead, aggressively keeps on searching.
+- The routing tables are poorly maintained. This can cause a search that should be logarithmic in the size of the network to be linear.
+
+#### Reachable
```

Comment:
this is an odd headline. maybe "Reachability"?

Reply:
Addressed in 73eaab8.